test/e2e/smoke-test/packages/cert-manager/1.5.3/cert-manager-test.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+
+# Checking package is installed or not
+tanzu package installed list | grep "cert-manager.community.tanzu.vmware.com"
+
+exitcode=$?
+
+if [ "${exitcode}" == 1 ] 
+then
+version=$(tanzu package available list cert-manager.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
+tanzu package install cert-manager --package-name cert-manager.community.tanzu.vmware.com --version "${version}"
+sleep 30s
+fi
+
+
+# Providing prerequisite 
+cat <<EOF | kubectl apply --filename -
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: self-signed
+spec:
+  selfSigned: {}
+EOF
+
+cat <<EOF | kubectl apply --filename -
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: private-ca
+spec:
+  ca:
+    secretName: private-ca
+EOF
```

Comment:
`private-ca` `Issuer` needs `private-ca` secret. Given that `private-ca` secret is created only after `private-ca` `Certificate` is created, I think it's better to put the `private-ca` `Certificate` yaml along with it's `kubectl apply` **before** the `private-ca` `Issuer` is created, basically following the order of creation as mentioned here - https://cert-manager.io/docs/configuration/selfsigned/#bootstrapping-ca-issuers , self-signed-issuer -> Bootstrap / create Certificate for CA -> Create issuer with CA Certificate